test/plugin/scenarios/jedis-scenario/bin/startup.sh

```diff
@@ -18,5 +18,5 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} -Dskywalking.plugin.jedis.trace_redis_parameters=true\
+java -Dskywalking.plugin.jedis.trace_redis_parameters=true -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT}\
```

Comment:
The problem here is not related to where you place the system property but _the space at the end of the line._

This is the reason you failed to run it locally and the CI reported.